packages/database/e2e/reference/child.e2e.js

```diff
@@ -38,7 +38,7 @@ describe('database().ref().child()', () => {
       return Promise.reject(new Error('Did not throw an Error.'));
     } catch (error) {
       error.message.should.containEql(
-        "'Paths must be non-empty strings and can't contain '.', '#', '$', '[', or ']''",
+        "'Paths must be non-empty strings and can't contain #, $, [, ], ' or ?'",
```

Comment:
the `'` wrapping around the double quoted string is redundant? E2E test seems fine with it though

Reply:
It executed (https://github.com/invertase/react-native-firebase/pull/4177/checks?check_run_id=1048435785#step:20:961) and passed, seems okay